app/lib/report_helper.rb

```diff
@@ -176,13 +176,13 @@ def external_report_info(pipeline_run_id, background_id, params)
     data
   end
 
-  def report_details(pipeline_run, background)
+  def report_details(pipeline_run, _background = nil)
+    # Provides some auxiliary information on pipeline_run.
+    # Not actually anything to do with report-specific data and does not use _background argument.
```

Comment:
there is always room for more cleanup, i suppose -- like getting rid of unused argument and renaming to closer match the meaning

but an improvement already